src/Builder/Stage/FactoryTrait.php

```diff
@@ -265,8 +265,8 @@ public static function fill(
      * Default: false.
      */
     public static function geoNear(
-        string $distanceField,
```

Comment:
The parameter was put first because required parameters must be before the optional ones. Changing the order of the parameters would be a breaking change one the library is released.

Reply:
I worry that the server being able to make such changes at will (where we might require a BC break to maintain consistency) is going to inevitably become a problem down the line.

Had you not caught this before a release, how could [PHPLIB-1609](https://jira.mongodb.org/browse/PHPLIB-1609) have been implemented?